### DISCUSSION TITLE
NextDNS error with DoH

### DISCUSSION THREAD

**Discussion by lordraiden** [AdguardTeam/AdGuardHome#2749] [Q&A] [answered]
### Prerequisites

- [Version: v0.105.1] I am running the latest version
- [Yes] I checked the documentation and found no answer
- [Yes] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
Version: v0.105.1
* **How did you install AdGuard Home:**
Official Docker Hub
* **How did you setup DNS configuration:**
![imagen](https://user-images.githubusercontent.com/17175395/108606933-9a51d700-73bd-11eb-8997-6bf8a5b7dfcc.png)

### Expected Behavior
To work

### Actual Behavior
I get this error when I click on test uptreams
![imagen](https://user-images.githubusercontent.com/17175395/108606948-b05f9780-73bd-11eb-9fa7-2e1f3a6fd98f.png)

The url has 6 characters after the /  lowcase and numbers
https://dns.nextdns.io/xxxxxx

- If I remove a single character, works (I dont get the error)
- If add a single random characters, works (I dont get the error)
- If add a replace any character for a random characters, works (I dont get the error)

It looks like it works anyway becase the requests reach nextdns but the error annoys me :) and it doesn't make any sense

### Additional Information
I have tried with 2 instances of Adguard Home docker, same result

**Comment by lordraiden** [comment]
I created a new profile so I got a differnet URL now it works

The old one already deleted is this one in case it can be reproduced in another installation
https://dns.nextdns.io/26c5d5

**Reply by ameshkov** [reply]
Tbh, this sounds like a NextDNS issue, not ours.

**Reply by lordraiden** [reply]
Yes, most probably, I think you can close it.

**Comment by ameshkov** [comment, marked as answer]
Probably, a NextDNS issue

**Comment by drcongo** [comment]
I have this issue too, freshly installed AdGuard on a Raspberry PI. The only NextDNS servers AdGuard let me enter are the IP4 addresses, and visiting my.nextdns.io tells me I'm not using NextDNS.

**Reply by drcongo** [reply]
Scratch that, just deleted it and reinstalled using the actual binary recommended for the Pi, and the very latest version, and all is working!
